contrib/google.golang.org/grpc/tags.go

```diff
@@ -7,6 +7,14 @@ package grpc
 
 // Tags used for gRPC
 const (
-	tagMethod = "grpc.method"
-	tagCode   = "grpc.code"
+	tagMethod     = "grpc.method"
+	tagCode       = "grpc.code"
+	tagMethodKind = "grpc.method.kind"
+)
+
+const (
+	methodKindUnary           = "unary"
+	methodKindClientStreaming = "client_streaming"
+	methodKindServerStreaming = "server_streaming"
+	methodKindBidiStreaming   = "bidi_streaming"
```

Comment:
Can we remove the `*ing` suffixes? The names are good, but I'm thinking a bit shorter wouldn't hurt and also wouldn't be wrong. No strong feelings here.